src/components/MemoryItem.js

```diff
@@ -2,39 +2,33 @@ import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 
 MemoryItem.propTypes = {
+  onDetail: PropTypes.func.isRequired,
   image: PropTypes.node,
   subtitle: PropTypes.string,
 }
 
-export default function MemoryItem({ image, subtitle }) {
+export default function MemoryItem({ image, subtitle, onDetail }) {
   return (
-    <ListWrapper>
-      <li>
-        <Image src={image} alt="" width="100%" />
-        <span>{subtitle}</span>
-      </li>
-    </ListWrapper>
+    <Wrapper onClick={() => onDetail(image)}>
+      <Image src={image} alt="" width="100%" />
+      <span>{subtitle}</span>
+    </Wrapper>
   )
 }
 
-const ListWrapper = styled.ul`
-  list-style-type: none;
+const Wrapper = styled.section`
+  background-color: hsl(60, 2%, 80%, 0.8);
   display: grid;
-  gap: 20px;
+  align-items: center;
+  gap: 5px;
   padding: 5px;
-
-  li {
-    background-color: hsl(60, 2%, 80%, 0.8);
-    display: grid;
-    align-items: center;
-    gap: 5px;
-    padding: 5px;
-    text-align: center;
-    color: #230707;
-    box-shadow: 2px 2px 4px grey;
-  }
+  text-align: center;
+  color: #230707;
```

Comment:
Please consider adding css variables in your GlobalStyle (in a `:root` selector).